gobblin-utility/src/main/java/gobblin/util/JobConfigurationUtils.java

```diff
@@ -78,10 +82,19 @@ public static void putStateIntoConfiguration(State state, Configuration configur
    * @param fileName the name of the file to load properties from
    * @return a new {@link Properties} instance
    */
-  public static Properties fileToProperties(String fileName) throws IOException, ConfigurationException {
-    Path filePath = new Path(fileName);
+  public static Properties fileToProperties(String fileName)
+      throws IOException, ConfigurationException, URISyntaxException {
+
     PropertiesConfiguration propsConfig = new PropertiesConfiguration();
-    propsConfig.load(filePath.getFileSystem(new Configuration()).open(filePath));
+    Configuration conf = new Configuration();
+    Path filePath = new Path(fileName);
+    URI fileURI = filePath.toUri();
+
+    if (fileURI.getScheme() == null && fileURI.getAuthority() == null) {
```

Comment:
Should this be `||` ? if either scheme or authority is null, local should be used